Process/Step/AbstractControllerStep.php

```diff
@@ -23,55 +24,53 @@ abstract class AbstractControllerStep extends Controller implements StepInterfac
 {
     /**
      * Step name in current scenario.
-     *
-     * @var string
      */
-    protected $name;
+    protected string $name;
 
     /**
      * {@inheritdoc}
      */
-    public function getName()
+    public function getName(): string
     {
         return $this->name;
     }
 
     /**
      * {@inheritdoc}
      */
-    public function setName($name)
+    public function setName($name): void
     {
         $this->name = $name;
     }
 
     /**
```

Comment:
I guess @inheritdoc is not needed anymore either

Reply:
fixed